examples/misc_sound.html

```diff
@@ -99,8 +96,10 @@
 				mesh1.position.set( -250, 30, 0 );
 				scene.add( mesh1 );
 
-				var sound1 = new THREE.Audio( listener );
-				sound1.load( 'sounds/358232_j_s_song.ogg' );
```

Comment:
Maybe we can keep `load` in `THREE.Audio` (and `THREE.PositionalAudio`)? Internally it will create a `THREE.AudioBuffer`. That way the API stays the same but we gain `.setBuffer()`.